openach/tasks.py

```diff
@@ -15,3 +21,24 @@ def example_task(x, y):  # pragma: no cover
     An example for reference.
     """
     return x + y
+
+@shared_task
```

Comment:
- Can you choose a method name that indicates that indicates that this is updating the title/description for a source?
- Indentation should use spaces instead of tabs
- Method is missing documentation, should probably link to the opengraph website in the documentation